packages/ui/src/ui/pages/navigation/Navigation/NavigationError/RequestPermission/RequestPermission.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import ypath from '../../../../../common/thor/ypath';
+import {getParentPath} from '../../../../../utils/navigation';
+import RequestPermissions from '../../../tabs/ACL/RequestPermissions/RequestPermissions';
+import RequestPermissionIsNotAllowed from './RequestPermissionIsNotAllowed';
+import {YTError} from '../../../../../../@types/types';
+
+type Props = {
+    error: YTError;
+    path?: string;
+    cluster: string;
+    block: (mix: string) => string;
```

Comment:
It is better to avoid passing of className-generators to another components, cause it is harder to control and refactor such usages. If you need specific styles for internals it is better to create corresponding block in the file.